WordPress/Classes/ViewRelated/Me/AccountSettingsViewController.swift

```diff
@@ -88,6 +87,10 @@ private class AccountSettingsController: SettingsController {
             action: presenter.push(editEmailAddress(settings, service: service))
         )
 
+        // If the primary site has no name, then show the URL.
+        var primarySiteName = settings.flatMap { service.primarySiteNameForSettings($0) }
+        primarySiteName = (primarySiteName == nil || primarySiteName == "") ? settings?.webAddress : primarySiteName
+
```

Comment:
If you prefer instead of `primarySiteName  == ""` you can do `primarySiteName.isEmpty`, or you can even simplify with `(primarySiteName?.isEmpty ?? true) ? .................`

Reply:
Hey @danielebogo . I used `primarySiteName == ""` because `primarySiteName` is an optional and I didn't want to unwrap it just for this test. However, your suggestion of `(primarySiteName?.isEmpty ?? true)` is _much_ better! It shall be changed. Thank you!